modules/options/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -485,6 +485,7 @@ final case class BuildOptions(
       addJvmTestRunner = isTests && addJvmTestRunner,
       addJmhDependencies = jmhOptions.addJmhDependencies,
       extraRepositories = finalRepositories,
+      keepResolution = internal.keepResolution,
```

Comment:
is there a reason why we want to drop the resolution?

Reply:
I don't recall the exact numbers, but I remember `coursier.Resolution` instances can be quite heavy in memory (possibly in several MB, if not much more when spark is involved). Better not keep them around when they're not needed.